src/main/java/com/backbase/ct/bbfuel/input/TransactionsReader.java

```diff
@@ -25,7 +25,20 @@ public TransactionsPostRequestBody loadSingle(String externalArrangementId) {
         return getRandomFromList(load(globalProperties.getString(CommonConstants.PROPERTY_TRANSACTIONS_DATA_JSON)))
                 .withId(UUID.randomUUID().toString())
                 .withArrangementId(externalArrangementId)
-                .withBookingDate(DateUtils.addDays(new Date(), generateRandomNumberInRange(-180, 0)))
+                .withBookingDate(DateUtils.addDays(new Date(), generateRandomNumberInRange(-180, 0)));
+    }
+
+    /**
+     * To be able to retrieve check images for a transaction, we need at least one of the transactions with
+     * with transactions id = TRANS0000000000001
+     * We need to set the date of the transaction to today so that we can easly find that transaction and test it.
+     */
+    public static TransactionsPostRequestBody loadSingleWithCheckImages(String externalArrangementId) {
+        TransactionsPostRequestBody defaultBody = this.loadSingle(externalArrangementId);
```

Comment:
the `loadSingle` actually loads one random transaction that is defined in the data file. Some transactions are not likely to be paid by cheque (e.g. hamburger) or can be categorized as "other income". Maybe it is better just to hardcode this transaction instead. I would expect the image for `TRANS0000000000001` not to change as well.